slides.Rmd

```diff
@@ -595,31 +672,34 @@ It is possible to customise plots created in R.
 - Change the colour and width of the bars
 
 ```{r,  eval=FALSE,echo=TRUE}
-barplot(counts, col = "blue", space = 1)
+bar_plot <- ggplot(counts, aes(x = day, y = count)) +
+  geom_bar(stat = "identity", fill = "blue", width = 0.5)
+
+bar_plot
 ```
 
 - Add titles
 
 ```{r,  eval=FALSE,echo=TRUE}
-barplot(counts,
-  xlab = "Admission Day", 
-  ylab = "Number of Admissions",
-  main = "Distribution of Admission Day"
-)
+bar_plot <- ggplot(counts, aes(x = day, y = count)) +
+  geom_bar(stat = "identity") +
+  xlab("Admission Day") +
+  ylab("Number of Admissions") +
+  ggtitle("Distribution of Admission Day")
+
+bar_plot
```

Comment:
This ran off the page for me

Reply:
It looks ok to me, could you send a screenshot so I can see what's happening? Or, if you have time, could you see if you can fix it? I'm on leave after today until Wednesday so won't be able to fix it before we meet.